### PR TITLE
updated drone.yml with staging and prod

### PR DESCRIPTION
**Pull request by Anilreddy999**
Please review

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -51,3 +51,63 @@ pipeline:
...
+      - export MATOMO_MYSQL_USER=$${STAGING_MATOMO_MYSQL_USER}
+      - export MATOMO_MYSQL_PASSWORD=$${STAGING_MATOMO_MYSQL_PASSWORD}
+      - export MATOMO_MYSQL_PORT=$${STAGING_MATOMO_MYSQL_PORT}
+      - kd --insecure-skip-tls-verify -f kube/service.yml
+      - kd --insecure-skip-tls-verify -f kube/network-policy.yml
+      - kd --insecure-skip-tls-verify -f kube/deployment.yml
+      - kd --insecure-skip-tls-verify -f kube/ingress.yml
+    when:
+      branch: master
+      event: staging
```
Comment (tomfitzherbert) [inline .drone.yml]: This should be:

```
event: deployment
environment: staging
```

File: .drone.yml [hunk 2]
```diff
@@ -51,3 +51,63 @@ pipeline:
...
+      - export MATOMO_MYSQL_USER=$${PROD_MATOMO_MYSQL_USER}
+      - export MATOMO_MYSQL_PASSWORD=$${PROD_MATOMO_MYSQL_PASSWORD}
+      - export MATOMO_MYSQL_PORT=$${PROD_MATOMO_MYSQL_PORT}
+      - kd --insecure-skip-tls-verify -f kube/service.yml
+      - kd --insecure-skip-tls-verify -f kube/network-policy.yml
+      - kd --insecure-skip-tls-verify -f kube/deployment.yml
+      - kd --insecure-skip-tls-verify -f kube/ingress.yml
+    when:
+      branch: master
+      event: production
```
Comment (tomfitzherbert) [inline .drone.yml]: This should be:

event: deployment
environment: production

File: .drone.yml (hunk 1 above)
Comment (Anilreddy999) [reply .drone.yml]: Updated

File: .drone.yml (hunk 2 above)
Comment (Anilreddy999) [reply .drone.yml]: Updated

**Pull request merged**